contracts/governance/Vesting/VestingRegistry.sol

```diff
@@ -52,7 +52,7 @@ contract VestingRegistry is Ownable {
     address public staking;
 
     /// @notice Fee sharing proxy.
-    address public feeSharingProxy;
+    address public feeSharingCollectorProxy;
```

Comment:
same here

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/482/commits/61d88fbfe3ceb4b74fa8b612479c397faca5c290)